Replace CodeSandbox embed with playground in tutorials (#109)
cruise-automation/webviz: docs/src/2.5.AddingInteractivity.mdx

```js
// use `defaultCameraState` to turn Worldview to uncontrolled component
<Worldview defaultCameraState={{ ...DEFAULT_CAMERA_STATE, distance: 160 }}>
```

Next, let's add a `clickedObjectIds` state to keep track of the clicked spheres. We'll also use this `clickedObjectIds` state to generate cube obstacles.

```js

> const [clickedObjectIds, setClickedObjectIds] = React.useState([]);

Reviewer: I think it's common to `import React, { useState }` and just use `useState` rather than `React.useState`

      id => id !== object.clickedObjectId
    );
    setClickedObjectIds(newClickedObjectIds);
  }}
>
  {obstacleMarkers}
</Cubes>
```

Now we can interact with the scene:

> - whenever we click on a sphere, a red cube will appear at the same clicked position

Reviewer: ```suggestion
- Whenever we click on a sphere, a red cube will appear in its place.
```

> - clicking a cube will make itself disappear

Reviewer: ```suggestion
- Clicking a cube will make it disappear.
```
